gbfs.md

```diff
@@ -468,6 +469,7 @@ Field Name | Required | Type | Defines
 \- `vehicle_type_id` <br/>*(added in v2.1-RC)* | Conditionally Required | ID | The vehicle_type_id of this vehicle as described in [vehicle_types.json](#vehicle_typesjson-added-in-v21-rc). This field is required if the [vehicle_types.json](#vehicle_typesjson-added-in-v21-rc) is defined.
 \- `last_reported` <br/>*(added in v2.1-RC)* | Optional | Timestamp | The last time this vehicle reported its status to the operator's backend.
 \- `current_range_meters` <br/>*(added in v2.1-RC)* | Conditionally Required | Non-negative float | If the corresponding vehicle_type definition for this vehicle has a motor, then this field is required. This value represents the furthest distance in meters that the vehicle can travel without recharging or refueling with the vehicle's current charge or fuel.
+\- `pricing_plan_id` | Optional | ID | The `plan_id` of the pricing plan this vehicle is eligible for as described in [system_pricing_plans.json](#system_pricing_plans.json). 
```

Comment:
I understand that the `pricing_plan_id` is at the vehicle type level (instead of the vehicle type at the pricing level) to allow for easier dynamic pricing based on location, but based on my comment from September 15th (about the same plan having different pricing based on the vehicle type), this would be confusing. 
For example, if vehicle ABC is eligible for plan 123 with a certain `per_min_princing` and vehicle XYZ is also eligible for plan 123 but with another `per_min_princing`. Both ABC and XYZ vehicles would be linked to the same pricing_plan_id=123, but there would actually be 2 objects with `plan_id`=123 in `system_pricing_plans` including different `per_min_princing` based on the vehicle type and no way to know which vehicle is linked to which of the `plan_id`=123 objects in this case (without data manipulation). 
I'm curious to know if we are the only ones with this particular case? 
(FYI the "vehicle_type_id" is still in the `system_pricing_plans.json` example)

Reply:
This would effectively be two different pricing plans, and therefore would have two separate plan IDs, for example plan ID 123-ABC and plan ID 123-XYZ, the public name for these can be the same, but in the back end they are different. As a general rule of thumb, even is the public plan name is the same (ex: one-way), if the prices are different, they should be treated as separate plans with separate plan IDs.